exercises/house/house.spec.js

```diff
@@ -0,0 +1,306 @@
+import House from './house';
+
+describe('House', () => {
+  test('verse one - the house that jack built', () => {
+    const house = new House();
```

Comment:
Since you're declaring a `new House()` in every test, you can pull that out into a `beforeEach`like so:

```javascript
let house;

beforeEach(()=> {
  house = new House();
}

test(...)

xtest(...)
```

Reply:
Good spot 👍 will update